modules/check_in/app/services/travel_claim/client.rb

```diff
@@ -158,6 +158,26 @@ def claims_data
       }
     end
 
+    def auth_url
+      if btsss_ssm_urls_enabled? && settings.auth_url_v2.present?
+        settings.auth_url_v2
+      else
+        settings.auth_url
+      end
+    end
+
+    def claims_url
+      if btsss_ssm_urls_enabled? && settings.claims_url_v2.present?
+        settings.claims_url_v2
+      else
+        settings.claims_url
+      end
+    end
+
+    def btsss_ssm_urls_enabled?
+      settings.mock || Flipper.enabled?('check_in_experience_travel_btsss_ssm_urls_enabled') || false
```

Comment:
What is `settings.mock`? I see `@settings = Settings.check_in.travel_reimbursement_api` but then I don't see `mock` in the config.   

```yaml
  travel_reimbursement_api:
    auth_url: https://login.microsoftonline.us
    auth_url_v2: https://internal-dsva-vagov-staging-fwdproxy-1821450725.us-gov-west-1.elb.amazonaws.com:4483/
    tenant_id: abcdefgh-1234-5678-12345-11e8b8ce491e
    claims_url: https://dev.integration.d365.va.gov
    claims_url_v2: https://internal-dsva-vagov-staging-fwdproxy-1821450725.us-gov-west-1.elb.amazonaws.com:4452/
    claims_base_path: EC/ClaimIngestSvc
    redis_token_expiry: 3540 # 59 minutes
    service_name: BTSSS-API
```

Reply:
My understanding is that we (CIE) use that to configure how to respond to CHIP, etc, whether to use betamocks or not. My thinking was that I wanted to check that rather than call Flipper. I probably should've called that out in the PR.

@randomsync can you confirm I'm not doing something off-the-rails here? 🙂

Reply:
yes, that's right. for this code, we followed a similar pattern as other service classes to have an option to enable the mock based on a setting value but ended up not needing to add it to config.

If we ever turn the mocks on using that config value, all requests will go through betamocks anyways. But for clarity for this PR, I think we can remove `settings.mock` from this line